frontend/src/components/ViewTrips/save-trip-modal.js

```diff
@@ -169,10 +239,10 @@ class SaveTripModal extends React.Component {
     db.collection(COLLECTION_TRIPS)
         .add(tripData)
         .then(docRef => {
-          console.log("Document written with ID: ", docRef.id);
+          console.log('Document written with ID: ', docRef.id);
```

Comment:
Do you intend to keep these for the successful case?

Reply:
I think its fine to keep this for now. Do you think it would be better to remove it?

Reply:
not sure